src/fuzzy-search.ts

```diff
@@ -38,7 +38,7 @@ export default class FuzzySearch {
 
     this.quickPick.show();
 
-    this.find(' ');
+    this.find('');
```

Comment:
What about only remove this initial find?

Reply:
This find also triggers the recent files search. I'm assuming it might be nice to have your recents when opening